apps/web/test/lib/getSchedule.test.ts

```diff
@@ -1255,5 +1259,59 @@ describe("getSchedule", () => {
         { dateString: plus3DateString }
       );
     });
+
+    test("getSchedule can get slots of org's member event type when orgSlug, eventTypeSlug passed as input", async () => {
+      const org = await createOrganization({ name: "acme", slug: "acme" });
+
+      const organizer = getOrganizer({
+        name: "Organizer",
+        email: "organizer@example.com",
+        id: 101,
+        // So, that it picks the first schedule from the list
+        defaultScheduleId: null,
+        organizationId: org.id,
+        // Has morning shift with some overlap with morning shift
+        schedules: [TestData.schedules.IstWorkHours],
+      });
+
+      const scenario = await createBookingScenario(
+        getScenarioData(
+          {
+            eventTypes: [
+              {
+                id: 1,
+
+                slotInterval: 45,
+                length: 45,
+                users: [
+                  {
+                    id: 101,
+                  },
+                ],
+              },
+            ],
+            organizer,
+          },
+          { id: org.id }
+        )
+      );
+
+      const { dateString: plus1DateString } = getDate({ dateIncrement: 1 });
+      const { dateString: plus2DateString } = getDate({ dateIncrement: 2 });
+
+      const getScheduleRes = await getSchedule({
+        input: {
+          eventTypeSlug: scenario.eventTypes[0]?.slug,
+          startTime: `${plus1DateString}T18:30:00.000Z`,
+          endTime: `${plus2DateString}T18:29:59.999Z`,
+          timeZone: "Europe/London",
+          isTeamEvent: false,
+          orgSlug: "acme",
+          usernameList: [organizer.username],
+        },
+      });
+
+      expect(getScheduleRes).toBeDefined();
```

Comment:
We should make real assertions about the data returned.

Reply:
Yeah but there’s no reason to not add at least a couple assertions about the schedule that is returned so we verify if completely works

Reply:
Got it